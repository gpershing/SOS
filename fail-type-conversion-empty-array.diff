diff: tests/fail-type-conversion-empty-array.err: No such file or directory
