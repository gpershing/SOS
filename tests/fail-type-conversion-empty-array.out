Cannot cast empty array to non-array type
